tests/test.yml

```diff
@@ -0,0 +1,7 @@
+parameters:
+  kapitan:
+    dependencies:
+      - type: https
+        source: https://github.com/vshn/espejo/releases/download/${espejo:git_tag}/espejo-crd.yaml
```

Comment:
What's exactly the problem you try to solve here?

Reply:
It's actually the line below (line 7). It's a problem of the working dir that commodore expects. Without it, I run into this:
```
Compile component espejo...
Updating Jsonnet libraries...
Compiling catalog...
Dependency https://github.com/vshn/espejo/releases/download/v0.3.1/espejo-crd.yaml: fetching now
Dependency https://github.com/vshn/espejo/releases/download/v0.3.1/espejo-crd.yaml: successfully fetched
Dependency https://github.com/vshn/espejo/releases/download/v0.3.1/espejo-crd.yaml: saved to /app/espejo/dependencies/espejo/manifests/v0.3.1/crds/espejo-crd.yaml


Compile error: espejo/manifests/v0.3.1/crds/ for target: espejo not found in search_paths: [PosixPath('/app/espejo/vendor'), './dependencies/', PosixPath('/app/espejo/vendor'), './', PosixPath('/usr/local/lib/python3.8/site-packages/commodore'), '/tmp/tmpgsxs0btx.kapitan']
make: *** [Makefile:49: compile] Error 1
```
When compiling, Kapitan expects the `manifests` dir right below `espejo`, but the `output_path` sets the downloaded manifest in `dependencies/espejo/manifests/...`

Reply:
Hmm, I see. This might need some changes in commodore, but good enough for now I guess 👍